docs/usage/quickstart.rst

```diff
@@ -3,112 +3,154 @@
-    node-03
+     .. code:: shell
 
-    [k8s-cluster:children]
-    kube-node
-    kube-master
+      $ cd ..
+      $ mkdir group_vars ; cd group_vars
 
-Make sure to change IP-addresses, usernames etc. according to your
-infrastructure.
+  - In group_vars, create a file named :file:`kube-node.yml`. This file
+    declares the default storage configuration on hosts in the *kube-node*
+    group—that is, hosts on which Pods shall be scheduled
 
-In a second file, called :file:`kube-node.yml` in a :file:`group_vars`
-subdirectory of our inventory, we declare how to setup storage (in the
-default configuration) on hosts in the *kube-node* group, i.e. hosts on which
-Pods will be scheduled:
+     .. code-block:: yaml
 
-.. code-block:: yaml
+         metal_k8s_lvm:
+           vgs:
+             kubevg:
+               drives: ['/dev/vdb']
 
-    metal_k8s_lvm:
-      vgs:
-        kubevg:
-          drives: ['/dev/vdb']
+    In this example, every *kube-node* host is assumed to have a disk available
+    as :file:`/dev/vdb` which can be used to set up Kubernetes
+    *PersistentVolumes*. For more information about storage, see
+    :doc:`../architecture/storage`.
 
-In the above, we assume every *kube-node* host has a disk available as
-:file:`/dev/vdb` which can be used to set up Kubernetes *PersistentVolumes*. For
-more information about storage, see :doc:`../architecture/storage`.
+Clone or Copy the MetalK8s Git Repo
+-----------------------------------
 
-Entering the MetalK8s Shell
----------------------------
-To easily install a supported version of Ansible and its dependencies, as well
-as some Kubernetes tools (:program:`kubectl` and :program:`helm`), we provide a
-:program:`make` target which installs these in a local environment. To enter this
-environment, run :command:`make shell` (this takes a couple of seconds on first
-run)::
+Return to the home subdirectory.
 
+.. code-block:: shell
+
+  $ cd
+
+Clone the MetalK8s project from GitHub (if you're building a new machine, you
+may also need to install Git).
+
+.. code-block:: shell
+
+   $ git clone https://github.com/scality/metal-k8s
+
+
+Enter the MetalK8s Shell
+------------------------
+
+To install a supported version of Ansible and its dependencies, along with some
+Kubernetes tools (:program:`kubectl` and :program:`helm`), Metal Kubernetes
+provides a :program:`make` target that installs these in a local environment.
+To enter this environment, run ``make shell`` (this takes a few seconds
+when first run).
+
+  .. code::
+
+    $ cd metal-k8s
     $ make shell
     Creating virtualenv...
     Installing Python dependencies...
     Downloading kubectl...
     Downloading Helm...
     Launching metal-k8s shell environment. Run 'exit' to quit.
-    (metal-k8s) $
+    (metal-k8s) [centos@node-01 metal-k8s]$
 
-Now we're all set to deploy a cluster::
+Now, you're all set to deploy a cluster::
 
     (metal-k8s) $ ansible-playbook -i inventory/quickstart-cluster -b metal-k8s.yml
```

Comment:
This wouldn't be correct if the inventory is created as done earlier in this document.

Reply:
This is also not clear to me. Still an issue?

Reply:
This doesn't work because
- First, the doc says to create an inventory in whatever directory
- Then, it tells to clone the repository
- Then, it tells to run `ansible-playbook` in that directory only created right before, referring to an inventory *inside that directory*, which is not / can't be where the inventory was created in the first place.